css/exercise_two/index.html

```diff
@@ -0,0 +1,361 @@
+        <div>
+          <img src="../../../img/quote_vector.png" alt="&quot;">
+        </div>
+        <hr class="mt-3 mb-3 border border-dark w-100">
+        <p class="fw-custom fs-5 mb-6 letter-spacing-negative lh-sm">
+          Muchísimas opciones concentradas en un solo lugar.  Poder comparar, elegir y gestionar todo desde la web es increíble.
+        </p>
+        <div class="d-flex gap-3">
+          <img src="../../../img/rootstrap_client.png" alt="Roostrap logo">
+          <div class="d-flex flex-column ml-2">
+            <p class="fw-medium fs-6 m-0">
+              Sandra De La Roche
+            </p>
+            <p class="fw-light fs-7 m-0">
+              Administration Specialist
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="container d-flex flex-column justify-content-between align-items-center my-8 gap-4 gap-lg-5">
+    <h2 class="fw-bolder fs-1 text-center lh-base blue-dark-text">
+      ¿Tenés un servicio o producto para ofrecer?
+    </h2>
+    <h6 class="fw-custom fs-5 text-center m-0 lh-sm d-none d-lg-flex">
+      Si ofreces servicios o productos para empresas, PlanIT es el lugar para vos.<br> Escribí tus datos y nos contactaremos contigo.
+    </h6>
+    <h4 class="fw-bold text-center m-0 lh-sm d-lg-none">
+      Si ofreces servicios o productos para empresas, PlanIT es el lugar para vos. Dejanos tus datos y nos contactaremos contigo a la brevedad.
+    </h4>
+    <div>
+      <button class="btn shadow rounded border-0 dark-blue-gradient text-white fs-6 fw-bold px-5 py-2 mt-4">
+      QUIERO SER PARTE
+    </button>
+    </div>
+  </div>
+  <div class="container-fluid bg-custom-blue rounded-top-5 mt-8">
+    <div class="container d-flex flex-column">
+      <div class="d-flex flex-column flex-lg-row justify-content-between mt-4 gap-5 mt-5 mt-lg-6">
+        <h1 class="fw-bolder fs-0 text-white lh-sm m-0">
+          ¿Listo para organizarte <span class="text-custom-blue-light">10 veces</span> más rápido?
+        </h1>
+        <div class="d-flex flex-column justify-content-between w-100">
+          <div class="mb-1 fs-6 fs-xl-5 fw-medium d-flex flex-column justify-content-between gap-3">
+            <input type="text" placeholder="Empresa" class="w-100 bg-transparent border-bottom border-white border-0 mb-5 placeholder-white text-white">
+            <input type="text" placeholder="Email" class="w-100 bg-transparent border-bottom border-white border-0 mb-5 placeholder-white text-white">
+            <input type="text" placeholder="Mensaje" class="w-100 bg-transparent border-bottom border-white border-0 mb-5 placeholder-white text-white">
+          </div>
+          <button class="btn shadow rounded bg-white border-0 mt-3 mb-6 mb-lg-0">
+            <span class="dark-blue-gradient text-transparent background-clip-text fs-6 fw-medium m-0 py-1">
+              ENVIAR
+            </span>
+          </button>
+        </div>
```

Comment:
i believe this div should be a form, as the expected result for this is to be able to send messages to the app.